src/main/resources/application.properties

```
@@ -1,8 +1,6 @@
 spring.application.name=BookStore
 spring.datasource.url=jdbc:mysql://localhost/book_store?serverTimezone=UTC
```

```suggestion
spring.datasource.url=jdbc:mysql://localhost:3306/book_store?serverTimezone=UTC
```

Also, you can get properties from .env file `spring.config.import=optional:file:.env[.properties]`
In this case you will have like: 
spring.datasource.username=${MYSQLDB_USER} etc